spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -1,10 +1,25 @@
 require 'spec_helper'
 require 'fortnox/api/repositories/context'
 require 'fortnox/api/repositories/invoice'
-require 'fortnox/api/models/invoice'
+require 'fortnox/api/repositories/examples/all_examples'
+require 'fortnox/api/repositories/examples/find_examples'
+require 'fortnox/api/repositories/examples/save_examples'
 
 describe Fortnox::API::Repository::Invoice do
   include_context 'repository context'
 
-  it_behaves_like 'repositories', Fortnox::API::Model::Invoice
+  it_behaves_like 'repositories'
+
+  include_examples '.all'
+
+  include_examples '.find', :document_number
+
+  include_examples '.save', :comments, 'Comments' do
```

Comment:
With all of the related comments implemented this would become:
```
include_examples '.save', :comments do
  let( :value ){ 'An invoice comment' }
  let( :updated_value ){ 'An updated invoice comment' }
end
```